Public/Src/Tools/SandboxedProcessExecutor/SandboxedProcessExecutor.dsc

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { NetFx } from "Sdk.BuildXL";
+import * as Managed from "Sdk.Managed";
+import * as BuildXLSdk from "Sdk.BuildXL";
+import * as DetoursServices from "BuildXL.Sandbox.Windows";
+
+namespace SandboxedProcessExecutor {
+
+    export declare const qualifier: BuildXLSdk.DefaultQualifier;
+
+    @@public
+    export const exe = BuildXLSdk.executable({
+        generateLogs: true,
+        allowUnsafeBlocks: true,
+        assemblyName: "SandboxedProcessExecutor",
+        rootNamespace: "BuildXL.SandboxedProcessExecutor",
+        skipDocumentationGeneration: true,

```

Comment:
>skipDocumentationGeneration [](start = 8, length = 27)

This should be false

Reply:
Will be removed in the next iteration.

---
In reply to: [276827074](https://github.com/Microsoft/BuildXL/pull/159#discussion_r276827074) [](ancestors = 276827074)